metagraph/src/common/vectors/vector_algorithm.hpp

```diff
@@ -233,11 +628,11 @@ void call_zeros(const Bitmap &vector,
 }
 
 template <class Callback>
-void call_zeros(const sdsl::bit_vector &vector,
-                uint64_t begin, uint64_t end,
-                Callback callback,
-                bool atomic,
-                int mo = __ATOMIC_SEQ_CST) {
+inline void call_zeros(const sdsl::bit_vector &vector,
+                       uint64_t begin, uint64_t end,
+                       Callback callback,
+                       bool atomic,
+                       int mo) {
```

Comment:
Don't put too many inlines, it might take longer to compile and make the binary larger.

This function executes a large block from `begin` to `end`.

Reply:
I'm not sure, but I think I at least need to make the initial declaration `inline`, otherwise it doesn't compile since it complains about having multiple instantiations.